app/presenters/special_users.rb

```diff
@@ -6,23 +6,38 @@ def self.special_users
     Setting.find_or_create_by(key: 'special_users').value
   end
 
+  # Checks if the user is a Special User of the given position
+  def self.is?(user, position)
+    SpecialUsers.special_users[position] == user.username
+  end
+
+  def self.set(role, username)
+    Setting.set_hash('special_users', role, username)
+  end
+
+  def self.remove(role)
+    users = Setting.find_or_create_by(key: 'special_users')
+    users.value.delete(role)
+    users.save
+  end
+
   def self.communications_manager
-    User.find_by(username: special_users[:communications_manager])
+    User.find_by(username: special_users['communications_manager'])
```

Comment:
These changes from symbol to string will change the existing behavior, because all the current (manually-set) values in the production database were done with symbols.